src/Compilers/Test/Core/TargetFrameworkUtil.cs

```diff
@@ -180,25 +179,28 @@ .. TestBase.WinRtRefs
         [
             TestBase.MinAsyncCorlibRef
         ];
-        public static ImmutableArray<MetadataReference> Mscorlib45ExtendedReferences =>
+
+        /*
+         * ⚠ Dev note ⚠: TestBase properties end here.
+         */
```

Comment:
This comment was moved up because the properties below no longer depend on `TestBase`